OpenCombine.py

```diff
@@ -0,0 +1,22 @@
+# To use `OpenCombine.py`, figure out its full path.
+# Let's say the full path is `~/projects/OpenCombine/OpenCombine.py`.
+# Then add the following statement to your `~/.lldbinit` file:
+#
+#     command script import ~/projects/OpenCombine/OpenCombine.py
```

Comment:
Can I ask to rename this file according to [PEP 8](https://www.python.org/dev/peps/pep-0008/#package-and-module-names)?

Something like `opencombine_lldb.py` so it's immediately clear that this has something to do with LLDB.

Reply:
I've renamed the file as requested. I updated the comments in the file itself and in `README.md`.